src/_gettsim/functions_loader.py

```diff
@@ -310,7 +340,8 @@ def _load_aggregation_combined_dict_from_strings(sources):
             aggregation_dicts_defined_in_module = [
                 obj
                 for name, obj in inspect.getmembers(out)
-                if isinstance(obj, dict) and name.startswith("aggregation_")
+                if isinstance(obj, dict)
+                and name.startswith(("aggregation_", "interpersonal_links_"))
                 # if _is_function_defined_in_module(func, out.__name__)
```

Comment:
```suggestion
```

Can this line go?

ValueError below has two typos in `aggregation_ dictionarys`